### DISCUSSION TITLE
Creality K1 - Side fan not adjustable?

### DISCUSSION THREAD

**Discussion by witgit** [Guilouz/Creality-Helper-Script-Wiki#513] [Issues]
Hello,
I have a problem with the side fan in my K1.
An error appeared on the display and the printer turned off. When turned on again, the side fan runs at maximum speed. I can't control it. I did a factory reset, upgrade, and downgrade of the firmware. Without result. What could be the reason?

https://drive.google.com/file/d/1oyveH_-ITW8H0RyG7dQSW-TDOkvliKgl/view?usp=sharing
https://drive.google.com/file/d/1p0x9_-PH3hvgHS9oosoyWFbwj7BdGt31/view?usp=sharing
https://drive.google.com/file/d/1p2yPpx8AMs1HISAyItcXlUzxZcPft4Ji/view?usp=sharing

**Comment by jamzilla** [comment]
Having the same issue with my K1 Max. Side fan turns on towards end of the print then won't turn off again. Factory reset using helper script, factory reset printer from screen. Still the same issues. Only thing that will turn the fan off is unplug it then plug it back in again until next print finished then it turns back onto full again then can't be controlled again until unplugged and replugged again.

@witgit Did you ever find a solution to the issue?

**Comment by winbard** [comment]
Hello,

i had the problem with the side fan taht it only start when it is over 70%.. And it make noise.
I bought a new one.  DELTA PRODUCT GROUPS / BFB1224VH
That work now. It is better and not so loud.
It seems that the orginal side fan was defect.
